src/guard/isAccessGrant.ts

```diff
@@ -30,8 +33,9 @@ export function isAccessGrant(
 ): vc is BaseAccessVcBody & AccessGrantBody {
   return (
     (vc as AccessGrantBody).credentialSubject.providedConsent !== undefined &&
-    (vc as AccessGrantBody).credentialSubject.providedConsent.hasStatus ===
-      GC_CONSENT_STATUS_EXPLICITLY_GIVEN &&
+    [GC_CONSENT_STATUS_EXPLICITLY_GIVEN, GC_CONSENT_STATUS_DENIED].includes(
+      (vc as AccessGrantBody).credentialSubject.providedConsent.hasStatus
+    ) &&
```

Comment:
What about access grants that are in other statuses? Perhaps it'd be better to be able to provide a range of statuses accepted when validating an AccessGrant?

Reply:
The intention here was to support only grants that are valid as per the shapes of the server-side consent service, which are currently only these two. Since this is a server-controlled thing, I'm not sure we want consumers to have freedom there.